src/lib/ElectrumClient.js

```diff
@@ -472,7 +472,7 @@ export default class Client {
  * @param {string} script ScriptPubKey in a hexadecimal format.
  * @return {string} Script hash as a hex string.
  */
-function scriptToHash(script) {
+export function scriptToHash(script) {
```

Comment:
I wonder if we rename this to `scriptToElectrumHash` for clarity--and maybe even make it a `static` method on the default export class?

Reply:
That's a good idea, I made it a static method inside `Client`:
https://github.com/keep-network/tbtc.js/pull/133/commits/6616442a4c858f9defd94459368acfcec81b6526